src/base/net/geoipmanager.cpp

```diff
@@ -99,24 +98,40 @@ void GeoIPManager::loadDatabase()
     QString error;
     m_geoIPDatabase = GeoIPDatabase::load(filepath, error);
     if (m_geoIPDatabase)
-        Logger::instance()->addMessage(tr("GeoIP database loaded. Type: %1. Build time: %2.")
+        Logger::instance()->addMessage(tr("IP database loaded. Type: %1. Build time: %2.")
```

Comment:
"GeoIP" is a trademark of MaxMind and since we don't use it's db anymore we shouldn't use the name either.
If you think that "IP database" isn't enough, can I propose "IP geolocation database" instead?

Reply:
>can I propose "IP geolocation database" instead?

Ok.